apps/web/src/server/app/inspector/inspector.controller.js

```diff
@@ -1,10 +1,79 @@
-import { findAppealById } from './inspector.service.js';
+// @ts-check
+
+import * as inspectorSession from './inspector-session.service.js';
+import * as inspectorService from './inspector.service.js';
 
 /** @typedef {import('@pins/inspector').Appeal} Appeal */
+/** @typedef {import('@pins/inspector').AppealOutcome} AppealOutcome */
+/** @typedef {import('@pins/inspector').AppealSummary} AppealSummary */
+/** @typedef {import('@pins/inspector').SiteVisitType} SiteVisitType */
+
+/**
+ * @typedef {Object} ViewDashboardRenderOptions
+ * @property {AppealSummary[]} appeals - A collection of appeals assigned to the user.
+ */
+
+/**
+ * View the dashboard listing the inspector's assigned appeals.
+ *
+ * @type {import('@pins/express').QueryHandler<{}, ViewDashboardRenderOptions>}
+ */
+export async function viewDashboard(_, response) {
+	const appeals = await inspectorService.findAllAssignedAppeals();
+
+	response.render('inspector/dashboard', { appeals });
+}
+
+/**
+ * @typedef {Object} ViewAvailableAppealsRenderOptions
+ * @property {AppealSummary[]} appeals - A collection of appeals available for
+ * assignment to the inspector.
+ * @property {number[]=} selectedappealIds - An array of appeal ids already
+ * selected in the page form.
+ */
+
+/**
+ * View a list of appeals available for assignment to the inspector.
+ *
+ * @type {import('@pins/express').QueryHandler<{}, ViewAvailableAppealsRenderOptions>}
+ */
+export async function viewAvailableAppeals(_, response) {
+	const appeals = await inspectorService.findAllUnassignedAppeals();
+
+	response.render('inspector/assign-appeals', { appeals });
+}
+
+/**
+ * @typedef {Object} AssignAvailableAppealsBody
+ * @property {number[]} appealIds - An array of appealIds to be assigned
+ */
+
+/**
+ * @typedef {Object} AssignAppealsSuccessRenderOptions
+ * @property {AppealSummary[]} appeals - The appeals successfully assigned to the user.
+ */
+
+/**
+ * @type {import('@pins/express').CommandHandler<{}, ViewAvailableAppealsRenderOptions | AssignAppealsSuccessRenderOptions, AssignAvailableAppealsBody>}
+ */
+export async function assignAvailableAppeals({ body }, response) {
+	const appeals = await inspectorService.findAllUnassignedAppeals();
+
+	if (response.locals.errors) {
```

Comment:
Is there a reason for storing the errors in `locals` rather than the body? If this approach is better we should update all the journeys to also use this approach to keep consistency.

Reply:
I suspected it was something like that, I'm not too familiar with express/backend in general though so wasn't sure.